### ISSUE TITLE
Upgrade script or common library

### ISSUE THREAD

**Issue by oktaal** [CentreForDigitalHumanities/cookiecutter-webapp-deluxe#59]
When maintaining multiple generated projects; it is laborious to sync updates made to components/services/etc in the template and in generated projects. I add some improvement to the generated output of project A, now I can (or maybe not) update this in the cookiecutter and then I may (or not) update projects B, C and D. Halfway through realizing a modification in project C is somewhat related and should be merged with the modification and I get to start all over. Argh.

Ideally it should be possible to update the template and then apply this update automatically to existing projects. For example using this project upgrader: [pypi/cookiecutter-project-upgrader](https://pypi.org/project/cookiecutter-project-upgrader/) (or perhaps similar tools exist). Another approach might be to have some kind of shared library which could then be imported instead of generated.

**Comment by jgonggrijp** [comment]
Yeah I think shared libraries are to be preferred whenever possible. Much more flexible and easier to synchronize. Something like cookiecutter-project-upgrader is certainly also interesting, though. In the end, there will always be some glue code that needs to be included in the cookiecutter itself.
